CppSamples/ChilliConnectCppDemo/ChilliConnect.cpp

```diff
@@ -0,0 +1,122 @@
+#include "ChilliConnect.h"
+
+ChilliConnect::ChilliConnect(const string gameToken)
+{
+	this->gameToken = gameToken;
+	this->httpSystem = unique_ptr<HttpSystem>(new HttpSystem());
+}
+
+ChilliConnect::~ChilliConnect()
+{
+}
+
+bool
+ChilliConnect::parse(const HttpResult &httpResult, ChilliConnectResponse & response, Json::Value & json)
+{
+	response.wasOk = (httpResult.code == 200);
+	response.rawBody = httpResult.body;
+
+	bool parsingSuccessful = false;
+	if (!httpResult.body.empty()) {
+		Json::Reader reader;
+		parsingSuccessful = reader.parse(httpResult.body, json);
+		if (!response.wasOk) {
+			response.errorCode = json["Code"].asInt();
+			response.errorMessage = json["Error"].asString();
+		}
+	}
+
+	return parsingSuccessful;
+}
+
+CreatePlayerResponse
+ChilliConnect::CreatePlayer() 
+{
+	//https://docs.chilliconnect.com/api/?system=http#api-PlayerAccounts-CreatePlayer
+
+	map<string, string> headers;
+	headers["Game-Token"] = "Vv7VANzImRtEUeiYaoz4lWKqB6t349iy";
+
+	HttpResult httpResult = this->httpSystem->MakePostRequest("connect.chilliconnect.com", "1.0/player/create", headers, "");
```

Comment:
That's a good point, but just wanting to show how to call Chilli for now, if we get some time later I agree it would be good to come back and make this a bit more robust. I'll stress in the readme too that this is not intended for production use.